src/cdk/coercion/array.ts

```diff
@@ -7,6 +7,6 @@
  */
 
 /** Wraps the provided value in an array, unless the provided value is an array. */
-export function coerceArray<T>(value: T | T[]): T[] {
+export function coerceArray<T>(value: T | readonly T[]): T[] {
```

Comment:
Maybe we can do something like this so that if you pass in a `ReadonlyArray`, you get a `ReadonlyArray` back?

```
export function coerceArray<T>(value: T | T[]): T[];
export function coerceArray<T>(value: T | ReadonlyArray<T>): ReadonlyArray<T>;
export function coerceArray<T>(value: T | T[]): T[] {
  return Array.isArray(value) ? value : [value];
}
```

Reply:
yes, the overload will be better, thanks. will update 👍